### PR TITLE
Sunset feature request vitamin

### PR DESCRIPTION
**Pull request by maruf12**
Changes:
- Delete section request vitamin in isoman page
- Redirect request vitamin to isoman
- Redirect page data to portal data jabar

Related:
https://app.clickup.com/t/860ph146m

### Evidence
title: Sunset feature request vitamin
project: Pikobar Website
participants: @maruf12 @naufalihsank @sandisunandar99

### CODE REVIEW

File: middleware/redirect.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      path: '/data',
+      to: 'statistic'
+    },
+    {
+      path: '/distribution-case',
+      to: 'statistic'
+    },
+    {
+      path: '/distribution-healthcare',
+      to: 'healtcare'
```
Comment (naufalihsank) [inline middleware/redirect.js:34]: is it really healtcare not healthcare ? 🤔

File: middleware/redirect.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      to: 'table-case'
+    }
+  ]
+
+  routeList.forEach((el) => {
+    if (route.path === el.path) {
+      switch (el.to) {
+        case 'table-case':
+          return redirect(process.env.URL_TABLE_CASE)
+        case 'healtcare':
```
Comment (naufalihsank) [inline middleware/redirect.js:47]: this one too

**Review by naufalihsank — APPROVED** [review]

File: middleware/redirect.js (hunk 1 above)
Comment (maruf12) [reply middleware/redirect.js:34]: @naufalihsank yap typo, thanks for your correction

File: middleware/redirect.js (hunk 2 above)
Comment (maruf12) [reply middleware/redirect.js:47]: @naufalihsank yap typo, thanks for your correction

**Pull request merged**